xla/service/float_normalization_test.cc

```diff
@@ -586,6 +586,67 @@ TEST_F(FloatNormalizationNoComputeSupportTest,
   EXPECT_EQ(ShapeUtil::GetSubshape(crs->shape(), {1}).element_type(), BF16);
 }
 
+TEST_F(FloatNormalizationNoComputeSupportTest,
+       NormalizationClonesSharedApplyAllReduceAndReduce) {
+  auto module = CreateNewVerifiedModule();
```

Comment:
I personally prefer to build the graph with text HLO. I find it more readable.
Do as you prefer.

Reply:
I also prefer that, but this file avoids that way of building modules. When in Rome...